src/docs/product/dashboards/custom-dashboards/index.mdx

```diff
@@ -17,18 +17,23 @@ Custom Dashboards are shared across your organization, and users can view each o
 ## Manage Dashboards
 When you have either a Business or Trial Plan, navigating to **Dashboards** takes you to a view where you can manage dashboards instead of the [default dashboard](/product/dashboards/). From here, you can:
 - Create a dashboard
+- Create a dashboard from a template
 - Duplicate a dashboard
 - Delete a dashboard
 
 ![Manage mode.](dashboard-management.png)
 
+### Create Dashboards
 Clicking "Create Dashboard" takes you to a new dashboard where you can immediately begin adding widgets. When you're creating a new dashboard or editing an existing one, the widgets are slightly grayed out. From here, you can:
 - Rename the dashboard
 - Add and remove widgets
 - Rearrange existing widgets
 
 ![Edit mode.](dashboard-edit.png)
 
+### Create Dashboards From Templates
+When the "Show Templates" toggle is enabled, there are several dashboard templates that come by default. Each template can be added directly to the list of dashboards by clicking "Add Dashboard" or can be previewed first by clicking "Preview". Templates must be added first before the resulting dashboard can be edited.
```

Comment:
Mostly just changing this to active rather than passive voice. 

```suggestion
When the "Show Templates" toggle is enabled, several default dashboard templates are displayed. You can add each template directly to the list of dashboards by clicking "Add Dashboard", or preview it by clicking "Preview". You have to add a template before you can edit the resulting dashboard. You can also duplicate or delete a dashboard created from a template (this does not delete the template). 
```